pkg/agent/agent.go

```diff
@@ -492,9 +500,19 @@ func (a *Agent) ProcessSnapshot(ctx context.Context, snapshot *snapshotTypes.Sna
 	}
 
 	if snapshot.Kubernetes != nil {
-		if a.podStore != nil {
-			snapshot.Kubernetes.Pods = a.podStore.GetPodsForServices(snapshot.Kubernetes.Services)
-			dlog.Debugf(ctx, "Found %d pods and %d services", len(snapshot.Kubernetes.Pods), len(snapshot.Kubernetes.Services))
+		if a.coreStore != nil {
+			if a.coreStore.podStore != nil {
+				snapshot.Kubernetes.Pods = a.coreStore.podStore.StateOfWorld()
+				dlog.Infof(ctx, "Found %d pods", len(snapshot.Kubernetes.Pods))
```

Comment:
maybe add this as a debug line? cuz that'll be a lotta logs. same for the below log lines

Reply:
Definitely. 
Done.